contracts/modules/ModuleFactory.sol

```diff
@@ -12,13 +12,14 @@ import "../libraries/VersionUtils.sol";
 contract ModuleFactory is IModuleFactory, Ownable {
 
     IERC20 public polyToken;
-    uint256 public setupCost;
     uint256 public usageCost;
     uint256 public monthlySubscriptionCost;
```

Comment:
I don't understand How it will increase the gas cost? could you share please share some errors with me that you got during the changes. 
If we are marking all other variables internal then for consistency make `usageCost` and `monthlySubscriptionCost` internal too and create the corresponding getters for them (This will also increase slight a bit of gas cost of deployment).

Reply:
I've got the following error for "npm run wintest" after removing getters:
Error: The contract code couldn't be stored, please check your gas amount.
Fixed by increased gas limit for development network from 7900000 to 8900000
Now I have another one
Error: Exceeds block gas limit

You can see changes in a separate branch - Remove-redundant-getters